chromiumoxide_types/src/lib.rs

```diff
@@ -48,9 +48,12 @@ pub trait Command: serde::ser::Serialize + Method {
     /// The type of the response this request triggers on the chromium server
     type Response: serde::de::DeserializeOwned + fmt::Debug;
 
-    /// deserialize the response from json
-    fn response_from_value(response: serde_json::Value) -> serde_json::Result<Self::Response> {
-        serde_json::from_value(response)
+    /// A helper method to make the response type inferenced without fullly qualifying response type that returns the identical value.
+    ///
+    /// For example: `let response = serde_json::from_str(res).map(CaptureSnapshotParams::infer_response)?;`
+    /// instead of `let response = serde_json::from_str::<<CaptureSnapshotParams as Command>::Response>(res)?;`
+    fn infer_response(res: Self::Response) -> Self::Response {
+        res
     }
```

Comment:
I turned the response_from_value into a generic helper function to use it [here](#diff-c81492ad19b277f815d99e4b236329469d8f29ae364f4b233a9b34e3b33bf930R215).

I don't believe this function must be as this, so I'm fine to revert the change or remove it.